CMakeLists.txt

```diff
@@ -54,7 +54,7 @@ set(PNET_MAX_CR                 2
 set(PNET_MAX_MODULES            5
   CACHE STRING "Per API. Should be > 1 to allow at least one I/O module")
 set(PNET_MAX_SUBMODULES         3
-  CACHE STRING "Per module (3 needed for DAP)")
+  CACHE STRING "Per slot (3 needed for DAP)")
```

Comment:
Maybe reword string for MAX_MODULES. "Max slots per API" or "Simultaneously plugged modules per API. " The relationship between slot/module subslot/submodule is not really clear.

Reply:
I plan to rename MAX_MODULES to MAX_SLOTS in a separate PR (after your  stuff is merged).